templates/index.html

```diff
@@ -50,8 +49,90 @@ <h1>Very fast server provisioning for your data centre</h1>
   </div>
 </section>
 
-<section class="p-strip">
+<section class="p-strip u-fixed-width">
+  <div class="p-strip is-shallow u-no-padding--top">
+    <h2>Data centre automation</h2>
+    <h4>Your next rack is a bare metal cloud.<br>Metal-As-A-Service (MAAS) provisioning with Windows, ESXi, Linux.</h4>
+  </div>
+  <ul class="p-list is-split">
+    <li class="p-list__item is-ticked">Bare metal cloud with on-demand servers</li>
+    <li class="p-list__item is-ticked">Remote edge cluster operations</li>
+    <li class="p-list__item is-ticked">Infrastructure monitoring and discovery</li>
+    <li class="p-list__item is-ticked">Ansible, Chef, Puppet, SALT, Juju integration</li>
+    <li class="p-list__item is-ticked">Super fast install from scratch</li>
+    <li class="p-list__item is-ticked">VMWare ESXi, Windows, CentOS, RHEL, Ubuntu</li>
+    <li class="p-list__item is-ticked">Custom images with pre-installed apps</li>
+    <li class="p-list__item is-ticked">Disk and network configuration</li>
+    <li class="p-list__item is-ticked">API-driven DHCP, DNS, PXE, IPAM</li>
+    <li class="p-list__item is-ticked">REST API for provisioning</li>
+    <li class="p-list__item is-ticked">LDAP user authentication</li>
+    <li class="p-list__item is-ticked">Role-based access control (RBAC)</li>
+    <li class="p-list__item is-ticked">Hardware testing and commissioning</li>
+  </ul>
+  <ul class="row u-no-margin--bottom p-strip is-shallow">
+    <li class="col-2 col-medium-1 col-small-1">
```

Comment:
This icons look tiny on medium screens:
![image](https://user-images.githubusercontent.com/1413534/94531531-98338400-0234-11eb-8597-b904385bda31.png)

Reply:
This is to match the size of the logos further down, otherwise they get way too big:
![image](https://user-images.githubusercontent.com/2741678/94541185-2f9dd480-023f-11eb-8a74-1e966768afb0.png)

They look small in the emulator but on a real device it looks much better. I'll attach a screenshot on an iphone in horisontal mode and on an ipad in a separate comment as  Ican't put an image here.

Reply:
It does look better on a real device. Just on shrunk down browsers that look bad. This is a design decision:
![image](https://user-images.githubusercontent.com/1413534/94544795-98874b80-0243-11eb-85fd-f809a20a91af.png)